### PR TITLE
Added some properties to the device entity

### PR DESCRIPTION
**Pull request by ceelias**

### CODE REVIEW

File: src/steps/devices/converter.ts [hunk 1]
```diff
@@ -33,6 +33,9 @@ export function createHostAgentEntity(device: Device): Entity {
         firewallEnabled: device['Firewall Enabled'] || undefined,
         fileVaultEnabled: device['FileVault Enabled'] || undefined,
         platform: device['OS Platform'] || undefined,
+        totalMemory: device['Total Memory (GB)'] || undefined,
+        totalDiskSpace: device['Total Disk Space (GB)'] || undefined,
+        lastOnline: device['Last Online'] || undefined,
```
Comment (VDubber) [inline src/steps/devices/converter.ts:38]: `totalMemory` and `totalDiskSpace` are both referenced in the api docs. Is lastOnline present on the response but not mentioned in the docs yet?

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ and this project adheres to
 
 ## [Unreleased]
 
+### Added
+
+- Added `totalMemory`, `totalDiskSpace`, and `lastOnline` to `Device` entity
```
Comment (austinkelleher) [inline CHANGELOG.md:13]: You don't have to change this, but I've been using a table for new properties. See: https://github.com/JupiterOne/graph-google-cloud/blob/main/CHANGELOG.md#added-1

**Review by austinkelleher — APPROVED** [review]

**Review by VDubber — APPROVED** [review]
Looks great! Added comment about missing property in docs.

File: src/steps/devices/converter.ts (hunk 1 above)
Comment (ceelias) [reply src/steps/devices/converter.ts:38]: Yea i appreciate you checking, the docs are not great. Here that property in the response:

Screen Shot 2022-02-07 at 3 24 30 PM

File: CHANGELOG.md (hunk 2 above)
Comment (ceelias) [reply CHANGELOG.md]: @austinkelleher I like that, updated to do that.

File: src/steps/devices/converter.ts (hunk 1 above)
Comment (ceelias) [reply src/steps/devices/converter.ts:38]: 💯  thanks for catching that. WIll fix

File: src/steps/devices/converter.ts [hunk 3]
```diff
@@ -33,6 +34,9 @@ export function createHostAgentEntity(device: Device): Entity {
         firewallEnabled: device['Firewall Enabled'] || undefined,
         fileVaultEnabled: device['FileVault Enabled'] || undefined,
         platform: device['OS Platform'] || undefined,
+        totalMemory: device['Total Memory (GB)'] || undefined,
+        totalDiskSpace: device['Total Disk Space (GB)'] || undefined,
+        lastOnline: parseTimePropertyValue(device['Last Online']) || undefined,
```
Comment (austinkelleher) [inline src/steps/devices/converter.ts:39]: Just FYI, `parseTimePropertyValue` already handles `undefined` values and will return `undefined` when `undefined` is supplied.

**Review by austinkelleher — APPROVED** [review]

**Review by austinkelleher — APPROVED** [review]

**Pull request merged**